test/integration/compile-test.coffee

```diff
@@ -7,7 +7,7 @@ createOriginSchema = require('../schemas/origin')
 {assert, compileFixture} = require('../utils')
 
 
-describe('compile() · all API description formats', ->
+describe.only('compile() · all API description formats', ->
```

Comment:
Forgotten `describe.only`?

Reply:
😱 

I really need to stop using it. Since I discovered `npm test -- --grep='all API description formats'` there is no point in using this and it only leads to mistakes like this. And when coverage/coveralls is broken, these mistakes can be very expensive 🙈